content/html/concepts/tags/terms/doctype/doctype.md

```diff
@@ -0,0 +1,31 @@
+---
+Title: '<doctype>' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
+Description: 'The <!DOCTYPE> tag tells the browser that the current file is in the HTML format.' # Required; ideally under 150 characters (used in search engine results and content previews)
+Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
+  - 'Web Development'
+  - 'Web Design'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'Browsers'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<!DOCTYPE>` tag tells the users browser that the current file is HTML and not another type such as JS or CSS
+
+## How to Use
```

Comment:
While I appreciate the brevity/simplicity of this entry, I do think it can be enhanced by adding a "Syntax" heading above this with just a code snippet of the tag like so:

## Syntax

```html
<!DOCTYPE html>
```

The "How to Use" heading can be changed to "Example" (the example below is great as is).